core/domain/exp_domain.py

```diff
@@ -375,11 +375,11 @@ def validate(self):
             raise utils.ValidationError(
                 'Invalid audio filename: %s' % self.filename)
         extension = self.filename[dot_index + 1:]
-        if extension not in feconf.ACCEPTED_AUDIO_EXTENSIONS:
+        if extension not in feconf.ACCEPTED_AUDIO_EXTENSIONS.keys():
```

Comment:
Just checking, does this change anything? Assuming its a dictionary it should work either way right?